class-lab/exercise4/TextboxWithCounter.js

```diff
@@ -0,0 +1,23 @@
+class TextboxWithCounter {
+  constructor(el) {
+    el.innerHTML = `
+    <p class="counter">0</p>
+    <textarea></textarea>`
+    this.counter = el.querySelector('.counter');
+    this.textarea = el.querySelector('textarea');
+    this.textarea.addEventListener('keyup',
+        this.updateCounter);
+  }
+
+  updateCounter = () => {
+    const len = this.textarea.value.length;
+    this.counter.innerHTML = len;
```

Comment:
It's considered a better practice to use `this.counter.textContent = len` instead of `innerHTML` when changing only the text

Reply:
thanks, just learned that there is a textContent property :-)